misc-tools/dj_make_chroot.in

```diff
@@ -396,6 +400,29 @@ if [ "$DISTRO" = 'Ubuntu' ]; then
     in_chroot "ln -s /bin/true /sbin/initctl"
 fi
 
+# Add additional repository lists to the chroot for upstream repos
+if [ -n "$REPOLISTS" ]; then
+    SPLIT_LIST=$(printf %s "$REPOLISTS" | tr ',' ' ')
+    for provided_repo in $SPLIT_LIST
+    do
+        cat "$provided_repo" >> "$CHROOTDIR/etc/apt/sources.list.d/upstream_chroot.list"
+        for filetype in arm gpg; do
+            if [ -f "${provided_repo}.${filetype}" ]; then
+                GPGFILENAME=$(mktemp -p "$CHROOTDIR/etc/apt/trusted.gpg.d" 'upstream_gpg_XXXXXX.gpg')
+                file_head=$(head -n1 "${provided_repo}.${filetype}")
+                if [ "-----BEGIN PGP ARMORED FILE-----" = "$file_head" ]; then
+                    gpg --dearmour "${provided_repo}.${filetype}"
```

Comment:
```suggestion
                    gpg --dearmor "${provided_repo}.${filetype}"
```
The gpg man-page only mentions the American-English spelling, so let's use that.